torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4471,19 +4489,23 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_inplace_autograd=False,
            sample_inputs_func=sample_inputs_kron),
     OpInfo('inner',
```

Comment:
Alphabetical order isn't being maintained, so I added the `tensordot` OpInfo beneath `inner`'s.